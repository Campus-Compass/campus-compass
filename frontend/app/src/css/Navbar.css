/* Navbar.css */
nav {
  background-color: #add8e6;
  padding: 10px;
    position: fixed;
  top: 0;
  width: 100%;
  z-index: 100; /* This ensures the navbar stays on top of other elements */
}

nav ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
  overflow: hidden;
}

nav li {
  float: left;
}

nav li a {
  display: block;
  color: white;
  text-align: center;
  padding: 14px 16px;
  text-decoration: none;
}

nav li a:hover {
  background-color: #6495ed;
}
