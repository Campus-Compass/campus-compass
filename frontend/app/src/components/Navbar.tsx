// Navbar.tsx
import React from 'react'
import { useNavigate } from 'react-router-dom'
import '../css/Navbar.css' // Import the CSS file
import AppBar from '@mui/material/AppBar'
import Box from '@mui/material/Box'
import Toolbar from '@mui/material/Toolbar'
import Button from '@mui/material/Button'
import logo from '../assets/logo.png'
import { useRoutesContext, IRoutesContext } from '../routes'
import { useTheme } from '@mui/material/styles'
import { useAuth } from '../auth/authProvider'

const settings = ['Profile', 'Logout']

const Navbar: React.FC = () => {
  const { getCurrentRouteName, getOtherNavbarRoutes } = useRoutesContext() as IRoutesContext
  const auth = useAuth()
  const pathname: string = getCurrentRouteName(window.location.pathname)
  const pages = getOtherNavbarRoutes(window.location.pathname, auth?.userRole ? auth.userRole : '')
  const theme = useTheme()
  const navigate = useNavigate()
  const [anchorElUser, setAnchorElUser] = React.useState<null | HTMLElement>(null)

  const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
    setAnchorElUser(event.currentTarget)
  }

  const handleCloseUserMenu = () => {
    setAnchorElUser(null)
  }

  return (
    <Box display={'flex'} mb={-2} p={2} gap={2}>
      <img onClick={() => navigate('/')} style={{ cursor: 'pointer' }} src={logo} width={50} />

      <AppBar
        position='static'
        color='secondary'
        sx={{
          flexShrink: '5',
          borderRadius: '20px',
          border: 'solid 3px',
          borderColor: theme.palette.primary.main
        }}
      >
        <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' }, gap: '40px' }}>
          <Button fullWidth key={pathname} sx={{ fontSize: '18px', borderRadius: '15px' }}>
            {pathname}
          </Button>
        </Box>
      </AppBar>

      <AppBar position='static' sx={{ flexShrink: '1', borderRadius: '20px' }}>
        <Toolbar disableGutters sx={{ height: '100%' }}>
          <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' }, height: '100%', width: '100%' }}>
            {pages.map((page: any) => (
              <Button
                fullWidth
                key={'navbtn-' + page.name}
                onClick={() => navigate(page.path)}
                color='secondary'
                sx={{ fontSize: '18px', borderRadius: '15px' }}
              >
                {page.name}
              </Button>
            ))}
          </Box>

          {/* <Box sx={{ flexGrow: 0 }}>
            <Tooltip title='Open settings'>
              <IconButton onClick={handleOpenUserMenu} sx={{ p: 0 }}>
                <Avatar alt='Remy Sharp' src='/static/images/avatar/2.jpg' />
              </IconButton>
            </Tooltip>
            <Menu
              sx={{ mt: '45px' }}
              id='menu-appbar'
              anchorEl={anchorElUser}
              anchorOrigin={{
                vertical: 'top',
                horizontal: 'right'
              }}
              keepMounted
              transformOrigin={{
                vertical: 'top',
                horizontal: 'right'
              }}
              open={Boolean(anchorElUser)}
              onClose={handleCloseUserMenu}
            >
              {settings.map((setting) => (
                <MenuItem key={setting} onClick={handleCloseUserMenu}>
                  <Typography textAlign='center'>{setting}</Typography>
                </MenuItem>
              ))}
            </Menu>
          </Box> */}
        </Toolbar>
      </AppBar>
    </Box>
  )
}

export default Navbar
