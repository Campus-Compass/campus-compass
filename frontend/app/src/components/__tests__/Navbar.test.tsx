import React from 'react'
import { render } from '@testing-library/react'
import Navbar from '../Navbar'
import { MemoryRouter } from 'react-router-dom'
import Routes from '../../routes'
import AuthProvider from '../../auth/authProvider'

test('renders Navbar correctly', () => {
  const { container } = render(
    <AuthProvider>
      <div className='App'>
        <Routes />
      </div>
    </AuthProvider>
  )
  expect(container).toMatchSnapshot()
})
