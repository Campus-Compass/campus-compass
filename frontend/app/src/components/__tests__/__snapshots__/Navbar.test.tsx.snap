// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navbar correctly 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="MuiBox-root css-fgqa3m"
    >
      <img
        src="logo.png"
        style="cursor: pointer;"
        width="50"
      />
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorSecondary MuiAppBar-positionStatic css-pdctmw-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiBox-root css-6kg5ta"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-fullWidth MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-fullWidth css-pzc2vi-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Home
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </header>
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-c6zvwl-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular css-131rfv7-MuiToolbar-root"
        >
          <div
            class="MuiBox-root css-pgilav"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorSecondary MuiButton-fullWidth MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorSecondary MuiButton-fullWidth css-17dqr3z-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Login
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorSecondary MuiButton-fullWidth MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorSecondary MuiButton-fullWidth css-17dqr3z-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Create Survey
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </header>
    </div>
    <div
      class="MuiBox-root css-1u5it4r"
    >
      <h2
        class="MuiTypography-root MuiTypography-h2 css-15afnto-MuiTypography-root"
      >
        Services Recommendation Survey
      </h2>
      <p
        class="MuiTypography-root MuiTypography-body2 css-1ypfif-MuiTypography-root"
      >
        Please answer the following so we may direct you to the proper services.
      </p>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg MuiContainer-disableGutters css-mek20u-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-0"
      >
        <form
          style="display: flex; flex-direction: column; align-items: center; padding-top: 15px;"
        >
          <div
            class="MuiBox-root css-lx4242"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            >
              Question 
              1
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              id="question"
            >
              Are you experiencing difficulties in your personal relationships?
            </p>
            <br />
            <div
              aria-label="survey"
              class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
              role="radiogroup"
            >
              <label
                class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              >
                <span
                  class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                >
                  <input
                    class="PrivateSwitchBase-input css-1m9pwf3"
                    name="survey"
                    type="radio"
                    value="1"
                  />
                  <span
                    class="css-hyxlzm"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                      data-testid="RadioButtonUncheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                      data-testid="RadioButtonCheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                >
                  Hell ye
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              >
                <span
                  class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                >
                  <input
                    class="PrivateSwitchBase-input css-1m9pwf3"
                    name="survey"
                    type="radio"
                    value="2"
                  />
                  <span
                    class="css-hyxlzm"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                      data-testid="RadioButtonUncheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                      data-testid="RadioButtonCheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                >
                  No
                </span>
              </label>
            </div>
          </div>
          <div
            class="MuiBox-root css-lx4242"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            >
              Question 
              2
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              id="question"
            >
              Are you an exchange student?
            </p>
            <br />
            <div
              aria-label="survey"
              class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
              role="radiogroup"
            >
              <label
                class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              >
                <span
                  class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                >
                  <input
                    class="PrivateSwitchBase-input css-1m9pwf3"
                    name="survey"
                    type="radio"
                    value="3"
                  />
                  <span
                    class="css-hyxlzm"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                      data-testid="RadioButtonUncheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                      data-testid="RadioButtonCheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                >
                  Yes
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              >
                <span
                  class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                >
                  <input
                    class="PrivateSwitchBase-input css-1m9pwf3"
                    name="survey"
                    type="radio"
                    value="4"
                  />
                  <span
                    class="css-hyxlzm"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                      data-testid="RadioButtonUncheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                      data-testid="RadioButtonCheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                >
                  No
                </span>
              </label>
            </div>
          </div>
          <div
            class="MuiBox-root css-lx4242"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            >
              Question 
              3
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              id="question"
            >
              Have you done your yearly health checkup?
            </p>
            <br />
            <div
              aria-label="survey"
              class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
              role="radiogroup"
            >
              <label
                class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              >
                <span
                  class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                >
                  <input
                    class="PrivateSwitchBase-input css-1m9pwf3"
                    name="survey"
                    type="radio"
                    value="5"
                  />
                  <span
                    class="css-hyxlzm"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                      data-testid="RadioButtonUncheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                      data-testid="RadioButtonCheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                >
                  Yes
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              >
                <span
                  class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                >
                  <input
                    class="PrivateSwitchBase-input css-1m9pwf3"
                    name="survey"
                    type="radio"
                    value="6"
                  />
                  <span
                    class="css-hyxlzm"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                      data-testid="RadioButtonUncheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                      data-testid="RadioButtonCheckedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                >
                  No
                </span>
              </label>
            </div>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-colorPrimary css-17oro8-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="submit"
          >
            Submit
            <span
              class="MuiButton-icon MuiButton-endIcon MuiButton-iconSizeLarge css-z4rqyd-MuiButton-endIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="SendIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M2.01 21 23 12 2.01 3 2 10l15 2-15 2z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </form>
      </div>
    </div>
  </div>
</div>
`;
